### PR TITLE
docs: add sql statement for load data

### PR DESCRIPTION
**Pull request by wshwsh12**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: sql-statements/sql-statement-load-data.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ aliases: ['/docs-cn/dev/reference/sql/statements/load-data/']
 
 ## 参数说明
 
-用户可以使用 `FIELDS` 参数来指定如何处理数据格式，使用 `FIELDS TERMINATED BY` 来指定每个数据的分隔符号，使用 `FIELDS ENCLOSED BY` 来指定消除数据的包围符号。如果用户希望以某个字符为结尾切分每行数据，可以使用 `LINES TERMINATED BY` 来指定行的终止符。
+用户可以使用 `LocalOpt` 参数来指定导入的数据文件位于客户端或者服务端。 目前 TiDB 只支持从客户端进行数据导入，因此在导入数据时 `LocalOpt` 应设置成 `Local`。
```
Comment (TomShawn) [inline sql-statements/sql-statement-load-data.md:20]:
```suggestion
用户可以使用 `LocalOpt` 参数来指定导入的数据文件位于客户端或者服务端。目前 TiDB 只支持从客户端进行数据导入，因此在导入数据时 `LocalOpt` 应设置成 `Local`。
```

File: sql-statements/sql-statement-load-data.md [hunk 2]
```diff
@@ -42,6 +44,8 @@ LINES TERMINATED BY '\n'
 
 用户可以通过 `IGNORE number LINES` 参数来忽略文件开始的 `number` 行，例如可以使用 `IGNORE 1 LINES` 来忽略文件的首行。
 
+另外，TiDB 目前对参数 `DuplicateOpt`、`CharsetOpt`、`LoadDataSetSpecOpt` 仅支持了语法解析。
```
Comment (TomShawn) [inline sql-statements/sql-statement-load-data.md:47]:
```suggestion
另外，TiDB 目前对参数 `DuplicateOpt`、`CharsetOpt`、`LoadDataSetSpecOpt` 仅支持语法解析。
```

**Comment by wshwsh12** [comment]
> 貌似语法图还是黄色的

https://github.com/pingcap/docs-cn/blob/docs-special-week/media/sqlgram/LoadDataStmt.png  这个文件被统一更改过了 已经是蓝的了 @SunRunAway

**Comment by SunRunAway** [comment]
@TomShawn Need one more LGTM.

**Pull request merged**